ui/app/styles/components/console-ui-panel.scss

```diff
@@ -5,12 +5,13 @@
 
 .console-ui-panel {
   background: var(--token-color-palette-neutral-700);
+  width: 100%;
```

Comment:
if you want, definitely up to you, but you could use a helper class "is-fullwidth".